### PR TITLE
Review and fix health/graphs metrics 

### PR DESCRIPTION
**Pull request by srikanthops**

### CODE REVIEW

File: enterprise-suite/es-monitor-api/static-rules.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@
   expr: avg by (device, instance) ((node_filesystem_free{device=~"/.*"} / node_filesystem_size * 100) and (node_filesystem_size > 0))
 
 - record: node_network_errors
```
Comment (dbrinegar) [inline enterprise-suite/es-monitor-api/static-rules.yml]: "node_network_error_rate"

Comment (dbrinegar) [reply enterprise-suite/es-monitor-api/static-rules.yml]: if a metric is a rate, the name should end in `_rate`

File: enterprise-suite/es-grafana/redis-exporter.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [
   {"graphName":"Redis Memory Used", "promQL":["redis_memory_used_bytes{workload=\"$workloadId\"}"]},
   {"graphName":"Commands Processed", "promQL":["irate(redis_commands_processed_total{workload=\"$workloadId\"}[1m])"]},
-  {"graphName":"Keys Evicted", "promQL":["redis_evicted_keys_total{workload=\"$workloadId\"}"]},
+  {"graphName":"Keys Evicted", "promQL":["irate(redis_evicted_keys_total{workload=\"$workloadId\"}[1m]"]},
```
Comment (dbrinegar) [inline enterprise-suite/es-grafana/redis-exporter.json]: Keys Evicted per Second

File: enterprise-suite/es-grafana/cassandra-exporter.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 [
-  {"graphName":"Storage", "promQL":["irate(cassandra_storage_load{workload=\"$workloadId\"}[1m])"]},
-  {"graphName":"Reads", "promQL":["irate(cassandra_clientrequest_latency{clientrequest='Read', workload=\"$workloadId\"}[1m])"]},
-  {"graphName":"Writes", "promQL":["irate(cassandra_clientrequest_latency{clientrequest='Write', workload=\"$workloadId\"}[1m])"]},
-  {"graphName":"Read Latency", "promQL":["irate(cassandra_clientrequest_totallatency{clientrequest='Read', workload=\"$workloadId\"}[1m])"]},
-  {"graphName":"Write Latency", "promQL":["irate(cassandra_clientrequest_totallatency{clientrequest='Write', workload=\"$workloadId\"}[1m])"]}
+  {"graphName":"Storage", "promQL":["cassandra_storage_load{workload=\"$workloadId\"}"]},
+  {"graphName":"Read Latency", "promQL":["cassandra_clientrequest_latency{clientrequest='Read', workload=\"$workloadId\"}"]},
+  {"graphName":"Write Latency", "promQL":["cassandra_clientrequest_latency{clientrequest='Write', workload=\"$workloadId\"}"]},
+  {"graphName":"Total Read Latency", "promQL":["cassandra_clientrequest_totallatency{clientrequest='Read', workload=\"$workloadId\"}"]},
```
Comment (dbrinegar) [inline enterprise-suite/es-grafana/cassandra-exporter.json]: These `totallatency` must be differentiated, monotonic graphs are too hard to read.  These are lifetime sums of latency.  These should be irate and "Latency per Second"

File: enterprise-suite/es-grafana/cassandra-exporter.json [hunk 4]
```diff
@@ -1,7 +1,7 @@
...
+  {"graphName":"Total Write Latency", "promQL":["cassandra_clientrequest_totallatency{clientrequest='Write', workload=\"$workloadId\"}"]}
 ]
```
Comment (dbrinegar) [inline enterprise-suite/es-grafana/cassandra-exporter.json]: whitespace problems:  missing newline at end of file as you can see in the github highlight above

File: enterprise-suite/es-monitor-api/static-rules.yml [hunk 5]
```diff
@@ -7,8 +7,8 @@
 - record: node_filesystem_free_percent
   expr: avg by (device, instance) ((node_filesystem_free{device=~"/.*"} / node_filesystem_size * 100) and (node_filesystem_size > 0))
 
-- record: node_network_errors
-  expr: sum by(instance) (node_network_transmit_errs + node_network_receive_errs + node_network_receive_drop + node_network_transmit_drop)
+- record: node_network_error_rate
```
Comment (dbrinegar) [inline enterprise-suite/es-monitor-api/static-rules.yml]: this name change must be accompanied by a change in the health rule that refers to the metric.  grep for "node_network_errors"

Comment (dbrinegar) [reply enterprise-suite/es-monitor-api/static-rules.yml]: you also need to update the smoke test, which looks for data with this metric name

File: enterprise-suite/es-monitor-api/bootstrap-monitors.json [hunk 6]
```diff
@@ -51,7 +51,7 @@
       "model": "threshold",
       "parameters": {
         "name": "node_network_errors",
-        "metric": "node_network_errors",
+        "metric": "node_network_rate",
```
Comment (dbrinegar) [inline enterprise-suite/es-monitor-api/bootstrap-monitors.json]: wrong name

**Comment by srikanthops** [comment]
smoke tests are fixed 
https://github.com/lightbend/k8s-explore/pull/146

File: enterprise-suite/es-monitor-api/bootstrap-monitors.json [hunk 7]
```diff
@@ -51,7 +51,7 @@
       "model": "threshold",
       "parameters": {
         "name": "node_network_errors",
-        "metric": "node_network_errors",
+        "metric": "node_network_errors_rate",
```
Comment (dbrinegar) [inline enterprise-suite/es-monitor-api/bootstrap-monitors.json]: this name isn't gramatical, as we don't say "errors rate" but rather we say "error rate"

Please change the name to "node_network_error_rate"

Comment (dbrinegar) [reply enterprise-suite/es-monitor-api/bootstrap-monitors.json]: I feel like I've been saying this over and over.  Please let me know if there is some kind of disagreement

Comment (srikanthops) [inline enterprise-suite/es-monitor-api/bootstrap-monitors.json]: There is no disagreement.. I debated this over..looked around and kept it inline with other names.

container_restarts_rate
kube_pod_container_restarts_rate

**Pull request merged**